File: 백준/Silver/2563. 색종이/색종이.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int[][] arr = new int[100][100];
        int N = sc.nextInt();
        int count = 0;
        
        for (int i = 0; i < N; i++) {
            int x = sc.nextInt();
            int y = sc.nextInt();
            
            for (int j = x; j < x + 10; j++) {
                for (int k = y; k < y + 10; k++) {
                    arr[j][k] = 1;
                }
            }
        }
        
        for (int i = 0; i < 100; i++) {
            for (int j = 0; j < 100; j++) {
                if(arr[i][j] == 1) {
                    count++;
                }
            }
        }
        System.out.println(count);
    }
}

File: 백준/Silver/2563. 색종이/색종이.cs
using System;

class Q2563
{
    static void Main(string[] args)
    {
        int[,] arr = new int[100,100];
        int N = int.Parse(Console.ReadLine());
        int count = 0;
        
        for (int i = 0; i < N; i++)
        {
            string[] S = Console.ReadLine().Split();
            int x = int.Parse(S[0]);
            int y = int.Parse(S[1]);
            
            for (int j = x; j < x + 10; j++)
            {
                for (int k = y; k < y + 10; k++)
                {
                    arr[j,k] = 1;
                }
            }
        }
        
        for (int i = 0; i < 100; i++)
        {
            for (int j = 0; j < 100; j++)
            {
                if(arr[i,j] == 1)
                {
                    count++;
                }
            }
        }
        
        Console.WriteLine(count);
    }
}